priv/cabbage/apps/itest/test/itest/payment_v2_test.exs

```diff
@@ -91,13 +95,48 @@ defmodule PaymentV2Test do
           %{amount: amount},
           %{bob: bob} = state do
     expecting_amount = Currency.to_wei(amount)
+
     balance = Client.get_exact_balance(bob.account, expecting_amount)["amount"]
 
-    assert_equal(expecting_amount, balance, "For Bob: #{bob.account}")
+    case expecting_amount do
+      0 -> assert(balance == nil, "Bob ETH amount is not 0")
+      _ -> assert_equal(expecting_amount, balance, "For Bob: #{bob.account}")
+    end
+
+    {:ok, state}
+  end
+
+  defwhen ~r/^Bob starts a standard exit with the payment v2 output$/, _, %{bob: bob} = state do
+    se = StandardExitClient.start_standard_exit(bob.account, @payment_v2_tx_type)
+    state = Map.put_new(state, :standard_exit, se)
+
+    {:ok, state}
+  end
+
+  defthen ~r/^Bob should no longer see the exiting utxo on the child chain$/, _, %{bob: bob} = state do
+    assert Itest.Poller.utxo_absent?(bob.account, state.standard_exit.utxo.utxo_pos)
+    assert Itest.Poller.exitable_utxo_absent?(bob.account, state.standard_exit.utxo.utxo_pos)
+    {:ok, state}
+  end
 
+  defwhen ~r/^Alice processes the standard exit on the child chain$/, _, state do
+    StandardExitClient.wait_and_process_standard_exit(state.standard_exit)
     {:ok, state}
   end
 
+  defthen ~r/^Alice should have "(?<amount>[^"]+)" ETH on the child chain after finality margin$/,
```

Comment:
Can we parametrize Alice - looks like it will work for any `entity`. Thank you 👍

Reply:
ok, add the step `Given Alice has an ethereum account` that generates the user account and put it into state instead!